tests/test_linalg.py

```diff
@@ -959,6 +959,19 @@ def test_solve(self, dtype):
 
         assert_allclose(expected, result, rtol=1e-06)
 
+    @pytest.mark.parametrize("dtype", get_all_dtypes(no_bool=True))
+    def test_solve_sat_6701(self, dtype):
```

Comment:
`SAT-6701` is not visible externally, so it will be unclear what the use case we are verifying here. Could you please rename the test?